sdk/identity/identity-broker/README.md

```diff
@@ -72,6 +72,18 @@ const credential = new InteractiveBrowserCredential({
 
 After calling `useIdentityPlugin`, the native broker plugin is registered to the `@azure/identity` package and will be available on the `InteractiveBrowserCredential` that supports WAM broker authentication. This credential has `brokerOptions` in the constructor options.
 
+**Notes**: As of `@azure/identity` version 4.11.0-beta.1, `DefaultAzureCredential` provides support to sign-in via the Windows Web Account Manager. Enable native broker in your program as follows:
+
+```ts snippet:using_plugins_dac
+import { useIdentityPlugin } from "@azure/identity";
+import { nativeBrokerPlugin } from "@azure/identity-broker";
+import { DefaultAzureCredential } from "../../identity/dist/commonjs/index.js";
```

Comment:
is the import from relative path file, the recommended way or should the import for DAC come from @azure/identity